### DISCUSSION TITLE
RX-V585

### DISCUSSION THREAD

**Discussion by EBME2** [mvdwetering/yamaha_ynca#351] [General]
RX-V585 Works as well, really happy I can ditch the Multicast. Great addon
![Screenshot 2024-12-15 182920](https://github.com/user-attachments/assets/8cb9bac7-93f8-4fbe-ba7d-a0663b219c40)

**Comment by mvdwetering** [comment]
Cool, thanks for the info. I added the RX-V585 to the list in the readme
